python/tvm/contrib/debugger/debug_executor.py

```diff
@@ -105,6 +106,7 @@ def __init__(self, module, device, graph_json_str, dump_root):
         self._profile = module["profile"]
         graph_executor.GraphModule.__init__(self, module)
         self._create_debug_env(graph_json_str, device)
+        self._device = device[0]
```

Comment:
i'm worried that the cpu device might not be first in the list. maybe we should just store a local flag e.g. `self._has_remote_device = any(d.device_type >= base.RPC_SESS_MASK for d in device)`

and use that?